contrib/pg_tde/src/catalog/tde_principal_key.c

```diff
@@ -746,8 +803,9 @@ get_principal_key_from_keyring(Oid dbOid, bool pushToCache)
 
 #ifndef FRONTEND
 	/* We don't store global space key in cache */
-	if (pushToCache && !TDEisInGlobalSpace(dbOid))
+	if (pushToCache && !(TDEisInGlobalSpace(dbOid)))
 	{
+		/* principalKey->keyInfo.databaseId = cacheOid; */
```

Comment:
What does the comment mean? Is it jsut a left-over? If not I think it needs more explanation.

Reply:
Leftover, I removed it